pom.xml

```diff
@@ -74,18 +75,45 @@
         </dependency>
     </dependencies>
 
+    <profiles>
+        <profile>
+            <id>doclint-java8-disable</id>
+            <activation>
+                <jdk>[1.8,)</jdk>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-javadoc-plugin</artifactId>
+                        <configuration>
+                            <additionalparam>-Xdoclint:none</additionalparam>
```

Comment:
Do not do this. Rather fix any actual Javadoc `error`s. Typically there are not many and it only takes a couple of minutes. `warning`s can be disregarded.